packages/shared/lib/core/layer-2/actions/getSelectedAccountLayer2Tokens.ts

```diff
@@ -1,6 +1,6 @@
 import { getSelectedAccountWrappedNativeTokens } from './getSelectedAccountWrappedNativeTokens'
 
-export async function getSelectedAccountLayer2Tokens(): Promise<{ amount: bigint; id: string }[]> {
+export async function getSelectedAccountLayer2Tokens(): Promise<unknown[]> {
```

Comment:
are the changes in this file some merge issues? 
i commented on some of matts PRs to revert these changes, and he did. So maybe these are some artifacts by branching off of that branch

Reply:
Yup, I fixed them.